src/IssueViz.Security/Taint/TaintIssuesSynchronizer.cs

```diff
@@ -101,5 +114,10 @@ private void ClearStore()
         {
             taintStore.Set(Enumerable.Empty<IAnalysisIssueVisualization>());
         }
+
+        private void SetHasTaintIssuesUIContext(bool hasTaintIssues)
```

Comment:
I kept reading it as 'SetHasTaintIssues', as in, a collection (set) has taint issues... Maybe `UpdateTaintUiContext` ?